### PR TITLE
[#154458914] adding ability to customize number of characters after the @ symbol

### PR DESCRIPTION
**Pull request by armandgray**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -70,26 +76,31 @@ export class MentionsTextInput extends Component {
...
+      } else {
+        this.closeSuggestionsPanel();
       }
     }
   }
 
   onChangeText(val) {
     this.props.onChangeText(val); // pass changed text back
     const lastChar = val.substr(val.length - 1);
-    const wordBoundry = (this.props.triggerLocation === 'new-word-only') ? this.previousChar.trim().length === 0 : true;
```
Comment (janeosaur) [inline index.js:87]: wordBoundary instead?

Comment (armandgray) [reply index.js:87]: is this a question?

Comment (armandgray) [reply index.js:87]: i think the answer is yes

**Review by janeosaur — COMMENTED** [review]
🇰🇷

**Comment by MattFoley** [comment]
💯

**Pull request merged**